app/models/policy.rb

```diff
@@ -17,7 +17,7 @@ class Policy < ActiveRecord::Base
   alias_attribute :status, :private
 
   def name_with_for
-    "for #{name}"
```

Comment:
This model method should not be changed.

If you need the name without "for" just call "#name".